### PR TITLE
Docs: Update Source Facebook Marketing docs

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
Updated the docs page for Facebook Marketing source.

## How
- Moved OSS-specific instructions to top of setup steps
- Merged OSS/Cloud steps where possible
- Updated descriptions of configurations, particularly for Custom insights inputs

### CODE REVIEW

File: docs/integrations/sources/facebook-marketing.md [hunk 1]
```diff
@@ -6,87 +6,94 @@ This page guides you through the process of setting up the Facebook Marketing so
...
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account, or navigate to your Airbyte Open Source dashboard.
 2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New source**.
-3. On the Set up the source page, select **Facebook Marketing** from the **Source type** dropdown.
-4. For Name, enter a name for your Facebook Marketing connector.
+3. Find and select **Facebook Marketing** from the list of available sources.
+4. For **Source name**, enter a name for your Facebook Marketing connector.
+5. To authenticate the connection:
+
+   <!-- env:cloud -->
+   **For Airbyte Cloud**: Click **Authenticate your account** to authorize your [Meta for Developers](https://developers.facebook.com/) account. Airbyte will authenticate the account you are already logged in to. Make sure you are logged into the right account.
```
Comment (sherifnada) [inline docs/integrations/sources/facebook-marketing.md:42]: why Meta For Developers account? do I need to be registered as a meta developer to use on Airbyte Cloud? That doesn't sound right

File: docs/integrations/sources/facebook-marketing.md [hunk 2]
```diff
@@ -6,87 +6,94 @@ This page guides you through the process of setting up the Facebook Marketing so
...
-6. For Account ID, enter your [Facebook Ad Account ID Number](https://www.facebook.com/business/help/1492627900875762): The Facebook Ad account ID to use when pulling data from the Facebook Marketing API. Open your Meta Ads Manager. The Ad account ID number is in the account dropdown menu or in your browser's address bar. See the [docs](https://www.facebook.com/business/help/1492627900875762) for more information.
-7. (Optional) Toggle the **Include Deleted** button to include data from deleted Campaigns, Ads, and AdSets.
+3. For **End Date**, use the provided datepicker, or enter the date programmatically in the `YYYY-MM-DDTHH:mm:ssZ` format. This is the date until which you'd like to replicate data for all Incremental streams. All data generated between the start date and this end date will be replicated. Not setting this option will result in always syncing the latest data.
```
Comment (sherifnada) [inline docs/integrations/sources/facebook-marketing.md:57]:
```suggestion
3. (Optional) For **End Date**, use the provided datepicker, or enter the date programmatically in the `YYYY-MM-DDTHH:mm:ssZ` format. This is the date until which you'd like to replicate data for all Incremental streams. All data generated between the start date and this end date will be replicated. Not setting this option will result in always syncing the latest data.
```

File: docs/integrations/sources/facebook-marketing.md [hunk 3]
```diff
@@ -145,15 +152,12 @@ You can segment the AdInsights table into parts based on the following informati
 For more information, see the [Facebook Insights API documentation.](https://developers.facebook.com/docs/marketing-api/reference/adgroup/insights/)
 
 :::note
-
-Pay attention, that not all fields (e.g. conversions, conversion_values) will be returned for AdInsights, see [docs](https://developers.facebook.com/docs/marketing-api/reference/ads-action-stats/).
-To get all fields You should use custom insights with **breakdowns**.
-
+Please note that not all fields, such as `conversions` and `conversion_values`, will be returned when accessing AdInsights. Refer to the [Facebook documentation](https://developers.facebook.com/docs/marketing-api/reference/ads-action-stats/) for more information. To retrieve all available fields, it is recommended to use Custom Insights and specify the necessary **breakdowns**.
```
Comment (sherifnada) [inline docs/integrations/sources/facebook-marketing.md:155]: What is `AdInsights` referring to here? the casing makes it sound like a specific thing like a class or specific stream. Should it be `ad insights`? 

Why would conversions not be returned? That seems weird given that conversions are an important part of ad analytics. Can we give the user a bit more info about when & why this happens? The link didn't provide more clarity for me personally

File: docs/integrations/sources/facebook-marketing.md [hunk 4]
```diff
@@ -6,87 +6,94 @@ This page guides you through the process of setting up the Facebook Marketing so
...
-**For Airbyte Open Source:**
+   To configure Custom Insights:
 
-To set up Facebook Marketing as a source in Airbyte Open Source:
+    1. For **Name**, enter a name for the insight. This will be used as the Airbyte stream name.
+    2. (Optional) For **Level**, enter the level of granularity for the data you want to pull from the Facebook Marketing API (`account`, `ad`, `adset`, `campaign`). Set to `ad` by default.
+    3. (Optional) For **Fields**, use the dropdown list to select the fields you want to pull from the Facebook Marketing API.
+    4. (Optional) For **Breakdowns**, use the dropdown list to select the breakdowns you want to configure.
+    5. (Optional) For **Action Breakdowns**, use the dropdown list to select the action breakdowns you want to configure.
+    6. (Optional) For **Action Report Time**, enter the action report time you want to configure. This value determines the timing used to report action statistics. For example, if a user sees an ad on Jan 1st but converts on Jan 2nd, this value will determine how the action is reported.
```
Comment (sherifnada) [inline docs/integrations/sources/facebook-marketing.md:78]: Great clarifications added here!

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/facebook-marketing.md (hunk 1 above)
Comment (ChristoGrab) [reply docs/integrations/sources/facebook-marketing.md]: Sorry bout that, got my Metas and Facebooks mixed up!

**Pull request merged**